modules/programs/neomutt.nix

```diff
@@ -196,7 +196,16 @@ let
 
   accountStr = account:
     with account;
-    ''
+    let
+      signature = if account.signature.showSignature == "none" then
+        "unset signature"
+      else if account.signature.command != null then
+        ''set signature = "${toString account.signature.command}|"''
```

Comment:
Are you sure that you want `toString` here? If you set `command = ./signature.sh` (i.e., a path literal) shouldn't this command point to the file in the Nix store?